gpuowl.cpp

```diff
@@ -166,9 +166,9 @@ bool load(Gpu *gpu, u32 E, u32 desiredBlockSize, int *outK, int *outBlockSize, i
   bool resOK = !loaded.res64 || res64 == loaded.res64;
   
   if (resOK && gpu->checkAndUpdate(loaded.blockSize)) {
-    log("OK loaded: %d/%d, blockSize %d, %016llx\n", loaded.k, E, loaded.blockSize, res64);
+    log("OK loaded %s: %d/%d, blockSize %d, %016llx\n", (globalCpuName.empty() ? "" : " ") + globalCpuName, loaded.k, E, loaded.blockSize, res64);
```

Comment:
But log() already puts the cpuName on the line always, together with the date. Why do you need it a second time?
Now it looks like this:
2018-08-24 21:54:48 vega0 OK loaded: 52432000/332345953, blockSize 400, 7d0ced3102346534
"vega0" being the cpuName.

Reply:
I'm not sure since I am with a previous version (3.5). The OK loaded line does not show the cpu name, and it does not show a timestamp.

Reply:
Probably, you changed this in a later version, so this patch only applies to version 3.6